File: 3737_count-subarrays-with-majority-element-i/csharp/main.cs
using System.Text.Json;

public class Program
{
    public static void Main()
    {
        var lines = Console.In.ReadToEnd().Split('\n');
        int[] nums = JsonSerializer.Deserialize<int[]>(lines[0]);
        int target = JsonSerializer.Deserialize<int>(lines[1]);
        var result = new Solution().CountMajoritySubarrays(nums, target);
        Console.WriteLine(JsonSerializer.Serialize(result));
    }
}

File: 3737_count-subarrays-with-majority-element-i/rust/main.rs
use std::io::{self, Read};

mod solution;
use solution::Solution;

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();
    let data: Vec<&str> = input.lines().collect();
    let nums: Vec<i32> = serde_json::from_str(data[0]).unwrap();
    let target: i32 = serde_json::from_str(data[1]).unwrap();
    let result = Solution::count_majority_subarrays(nums, target);
    println!("{}", serde_json::to_string(&result).unwrap());
}
